src/databrowser.js

```diff
@@ -38,6 +26,12 @@ define([
                 tree = this.$f.find(".fd-tree"),
                 pane = this.$f.find(".fd-accessory-pane"),
                 head, headHeight, searchBar, paneRatio;
+            vellum.data.core.databrowser = {
+                dataHashtags: {},
+                dataHashtagTransformations: {},
+                errorContainer: pane,
```

Comment:
I suppose it could be, but I'm not really interested in changing that in this PR. Doesn't seem too high priority.